cmd/monaco/v2/deploy/deploy.go

```diff
@@ -86,30 +86,49 @@ func Deploy(fs afero.Fs, deploymentManifestPath string, specificEnvironment stri
 		return errors.New("error while loading projects")
 	}
 
-	if specificProject != "" {
-		filtered, err := filterProjectIdsByName(projects, []string{specificProject})
+	err, projects = checkProjectsToDeploy(specificProject, projects, environmentNames)
```

Comment:
Since the method loads projects and not checks them,
could we rename it to `loadProjectsToDeploy` please?